clusterloader2/pkg/measurement/common/network/network_performance_measurement.go

```diff
@@ -396,3 +402,21 @@ func (npm *networkPerformanceMeasurement) getMetricData(data *testResultSummary,
 	data.DataItems = append(data.DataItems, metricDataItem)
 	klog.V(3).Infof("TestResultSummary: %v", data)
 }
+
+func (npm *networkPerformanceMeasurement) waitForMetricsFromPods() {
+	bufferTime := time.Duration(math.Max(10, float64(npm.numberOfClients)*podFactor)) * time.Second
```

Comment:
Let's just inline the "0.1" value here (the podFactor name isn't meaningful.

Reply:
modified